Makefile

```diff
@@ -12,6 +12,7 @@ TEST_RD_URL="redis://$(TEST_RD_USER)@$(TEST_RD_HOSTNAME):6379/$(TEST_RD_NAME)"
 PG_FIX_CANDIDATES=./protos/node/node.pb.go \
 			./protos/status/status.pb.go \
 			./protos/escrow/escrow.pb.go \
+			./protos/guard/guard.pb.go \
```

Comment:
you don't need to add this line, if guard's not using go-pg, since this is to fix tableName

Reply:
I know, original think is add it so that in the future if we want to add pg support to guard, we will not forget to add it back.